docs/install/managed-cluster.md

```
@@ -0,0 +1,922 @@
+
+**Critical:** The `planeID` in the DataPlane CR must match the `clusterAgent.planeId` Helm value.
+
+### Step 4: Install OpenChoreo Build Plane
+
+The Build Plane enables CI/CD capabilities for building container images and deploys a container registry.
+
+**Install Build Plane with Registry Configuration:**
+
+For managed Kubernetes, you have two options for the container registry:
+
+**Option 1: Use OpenChoreo's Built-in Registry (Exploration/Development)**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)